### PR TITLE
Rename commands

### PR DESCRIPTION
**Pull request by ovidharshini**
Directly addresses issue #133

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -60,23 +60,23 @@ You can also generate a PDF payslip for your employees to refer to.
...
-| `exit`      | `exit`                                                                                   | NA                                                                                                      |
-| `help`      | `help`                                                                                   | NA                                                                                                      |
+| Command        | Format                                                                                        | Examples                                                                                                |
+|----------------|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| `add`          | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS r/RATE [t/TAG]...​`                              | `add n/Nicole Tan p/99338558 e/nicole@stffhub.org  a/1 Tech Drive, S138572 r/37.50 t/Hardware t/Senior` |
```
Comment (zhongfu) [inline docs/UserGuide.md:65]: should this also be `personadd` for consistency?

**Review by zhongfu — COMMENTED** [review]
i'm gonna assume you/your IDE caught everything

looks fine, except i'm not quite sure about the person add command bit

File: docs/UserGuide.md (hunk 1 above)
Comment (ovidharshini) [reply docs/UserGuide.md:65]: Good point, I had the same consideration. Currently, the command for adding a job is `job`, so `person` or `personadd` (which also implies changing `job` to `jobadd`) seemed natural, but this is a command that would potentially be used a lot? `personadd`/`jobadd` seemed like a mouthful to type out in comparison to `add`, which is why I let it be.

I guess the added consistency is more relevant though; I'll push a commit to account for this

Comment (ovidharshini) [reply docs/UserGuide.md]: Changes made:
`add` -> `personadd`
`job` -> `add`

**Review by zhongfu — APPROVED** [review]
lgtm

**Pull request merged**